src/Player.tsx

```diff
@@ -67,10 +67,14 @@ const Player: Player = React.forwardRef((props, ref) => {
     return null;
   }
 
+  // Filter out ReactPlayer-specific event handlers to prevent them from being passed down
+  // to the underlying HTML video element, which causes React warnings about unknown
+  // event handler properties
   const eventProps: Record<string, EventListenerOrEventListenerObject> = {};
+  const reactPlayerEventHandlers = ['onReady', 'onStart', 'onPlay', 'onPause', 'onEnded', 'onLoadStart'];
```

Comment:
```suggestion
  const reactPlayerEventHandlers = ['onReady', 'onStart'];
```

`'onPlay', 'onPause', 'onEnded', 'onLoadStart'` are valid HTMLMediaElement events